flymake-vale.el

```diff
@@ -150,24 +178,24 @@ Passing the results and source BUF to CALLBACK."
   (when (process-live-p flymake-vale--proc)
     (kill-process flymake-vale--proc))
   (let* ((process-connection-type nil)
+         (callback flymake-vale--report-fnc)
+         (buf (current-buffer))
          (proc (apply #'start-process
                       "flymake-vale-process"
                       flymake-vale-output-buffer
                       flymake-vale-program
                       "--output"
                       "JSON"
-                      flymake-vale-program-args)))
+                      (flymake-vale--build-args))))
     (setq flymake-vale--proc proc)
-    (let ((callback flymake-vale--report-fnc)
-          (buf (current-buffer)))
-      (set-process-sentinel
-       proc
-       #'(lambda (p event)
-           (when (flymake-vale--normal-completion? event)
-             (if (eq proc flymake-vale--proc)
-                 (flymake-vale--handle-finished callback buf)
-               (flymake-log :warning "Canceling obsolete check %s"
-                            proc))))))
+    (set-process-sentinel
+     proc
+     #'(lambda (p event)
+         (when (flymake-vale--normal-completion? event)
+           (if (eq proc flymake-vale--proc)
+               (flymake-vale--handle-finished callback buf)
+             (flymake-log :warning "Canceling obsolete check %s"
+                          proc)))))
```

Comment:
I don't believe this is part of your commit

Reply:
You're right! I will force-push a rebased version without this bit.